explorer/models/taskNode.ts

```diff
@@ -55,12 +55,11 @@ export class TaskRootNode extends NodeBase {
             tasks = await client.tasks.list(resourceGroup, element.registry.name);
             if (tasks.length === 0) {
                 const learnHow: vscode.MessageItem = { title: "Learn How to Create Build Tasks" };
-                vscode.window.showInformationMessage(`You do not have any Tasks in the registry '${element.registry.name}'.`, learnHow).then(val => {
-                    if (val === learnHow) {
-                        // tslint:disable-next-line:no-unsafe-any
-                        opn('https://aka.ms/acr/task');
-                    }
-                })
+                let response = await vscode.window.showInformationMessage(`You do not have any Tasks in the registry '${element.registry.name}'.`, learnHow)
```

Comment:
Hmm I think this should still use a `then` (and maybe add a comment like `// don't wait` to make it more clear that this was intentional). Awaiting `showInformationMessage` means it won't return until the user dismisses the notification which might not happen for a long time if the user ignores the notification. Specifically - I wonder if this results in a loading "spinner" in the tree while we await